### PR TITLE
fix(config):ensure validatejs custom config is passed from validator.

### PR DESCRIPTION
**Pull request by apawsey**
Fix for #48.  Added custom configuration property to all methods on validator, and ensured config is passed through to validation rule.

### CODE REVIEW

File: src/validator.js [hunk 1]
```diff
@@ -32,28 +32,28 @@ export class Validator {
...
-  presence() {
-    this.config.addRule(this.currentProperty, ValidationRule.presence());
+  presence(configuration) {
+    this.config.addRule(this.currentProperty, ValidationRule.presence(configuration));
     return this;
   }
-  required() {
-    this.config.addRule(this.currentProperty, ValidationRule.presence());
+  required(configuration) {
+    this.config.addRule(this.currentProperty, ValidationRule.presenceconfiguration());
```
Comment (PWKad) [inline src/validator.js]: I think this is an invalid line.

File: src/validator.js [hunk 2]
```diff
@@ -32,28 +32,28 @@ export class Validator {
...
-  numericality() {
-    this.config.addRule(this.currentProperty, ValidationRule.numericality());
+  numericality(configuration) {
+    this.config.addRule(this.currentProperty, ValidationRule.numericality(configuration));
     return this;
   }
-  date() {
-    this.config.addRule(this.currentProperty, ValidationRule.date());
+  date(configuration) {
+    this.config.addRule(this.currentProperty, ValidationRule.dateconfiguration());
```
Comment (PWKad) [inline src/validator.js]: This one also

File: src/validation-renderer.js [hunk 3]
```diff
@@ -2,19 +2,18 @@ import {DOM} from 'aurelia-pal';
...
+    this.unrenderErrors(node);
     if (relevantErrors.length) {
       node.parentElement.classList.add('has-error');
       relevantErrors.forEach(error => {
         if (node.parentElement.textContent.indexOf(error.message) === -1) {
           let errorMessageHelper = DOM.createElement('span');
           let errorMessageNode = DOM.createTextNode(error.message);
           errorMessageHelper.appendChild(errorMessageNode);
-          errorMessageHelper.classList.add('help-block');
+          errorMessageHelper.classList.add('help-block', 'au-validation');
```
Comment (PWKad) [inline src/validation-renderer.js]: I think this change is also in your other PR as well can you back the changes out for this one and I can rebase / merge them back on top of master while merging?  You can do something like this if you aren't too familiar with GitHub - 

```
$ git reset HEAD~1
$ git checkout -- src/validation-renderer.js
$ git add .
$ git commit
```

**Comment by apawsey** [comment]
Yeah I don't know what happened with all of this, I got the branches mixed up, and made a mess of the files.  Those broken lines I thought I resolved, but then I reverted stuff and merged something... and I dunno (sigh) :smile:

First time I've actually contributed PR's - can you tell??

Let me try and fix it all, and resubmit it again

**Pull request closed without merging**